src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -10,13 +10,15 @@
 
 
 class JsonSummeryTests(TransactionTestCase):
```

Comment:
```suggestion
class JsonSummaryTests(TestCase):
```

I ran this locally and don't see why it would need a `TransactionTestCase` - those are much slower (2.505s vs. 0.082s on my system!)

And you can also fix the spelling mistake :)